boards/arm64/qemu_cortex_a55/qemu_cortex_a55_smp.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 NXP
+
+identifier: qemu_cortex_a55_smp
+name: QEMU Emulation for Cortex-A55 SMP
+type: qemu
+simulation: qemu
+arch: arm64
+toolchain:
+  - zephyr
+  - cross-compile
+ram: 128
+testing:
+  default: true
```

Comment:
Does this platform get us anything in CI that isn't convered by qemu_cortex_a53?